api/v1alpha1/openapi.yaml

```
@@ -4665,8 +4665,11 @@ components:
       description: EnrollmentServiceService contains information about connecting to a Flight Control enrollment service.
       properties:
         certificate-authority-data:
-          type: string
-          description: CertificateAuthorityData contains PEM-encoded certificate authority certificates.
+                  type: string
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 18

(indentation)